deploy/helm/flightctl/templates/flightctl-db-migration-job.yaml

```
@@ -28,14 +28,16 @@
-  "hookDeletePolicy" "before-hook-creation,hook-succeeded"
+  "hookDeletePolicy" "before-hook-creation,hook-succeeded,hook-failed"
   "isDryRun" false
 ) }}
 {{- else if .Release.IsInstall }}
 {{- include "flightctl.dbMigrationJob" (dict
   "context" $root
   "name" $baseName
-  "hooks" (list)
+  "hooks" (list "post-install")
+  "hookWeight" "10"
+  "hookDeletePolicy" "before-hook-creation,hook-succeeded,hook-failed"
```

YAMLlint (1.37.1)

[warning] too few spaces after comma

(commas)